app/controllers/api/v0x1/tasks_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V0x1
+    class TasksController < Api::V0x0::TasksController
```

Comment:
Could you add this to the `V0` base `TasksController` instead of just to `v0x1`?  We might need to update the swagger for `v0.0.1` to add patch for tasks but that shouldn't be that bad.

Reply:
Brandon didn't seem to want this when I made the original update PR #137. Specifically [this comment](https://github.com/ManageIQ/topological_inventory-api/pull/137#discussion_r260426103) makes me think we don't want to make any more changes to `v0.0.1`.

Reply:
Hm okay, well we're going to deprecate that soon anyway so we'll just have to move this from v0x1 to v0 then